### PR TITLE
Implemented IBM i Service Functions and Tagged Settings

### PR DESCRIPTION
**Pull request by Nikhil-Ashoka**
- Under Logs and services: https://jsw.ibm.com/browse/PFEBMC-1106
- Under Dumps: https://jsw.ibm.com/browse/PFEBMC-1848
- Under Server power operations: https://jsw.ibm.com/browse/PFEBMC-1849

### CODE REVIEW

File: src/views/Operations/ServerPowerOperations/BootSettings.vue [hunk 1]
```diff
@@ -75,6 +94,7 @@ export default {
       this.isLinuxKvmValid = value;
     },
     handleSubmit() {
+      console.log('In Submit');
```
Comment (Renuka9527) [inline src/views/Operations/ServerPowerOperations/BootSettings.vue]: remove console log

Comment (Nikhil-Ashoka) [reply src/views/Operations/ServerPowerOperations/BootSettings.vue]: Okay

**Comment by Nikhil-Ashoka** [comment]
[Uploading 1KW_GUI.docx…]()

**Comment by Nikhil-Ashoka** [comment]
[1KW_GUI.docx](https://github.com/ibm-openbmc/webui-vue/files/14190378/1KW_GUI.docx)

**Comment by Nikhil-Ashoka** [comment]
[1KW_GUI_review.docx](https://github.com/ibm-openbmc/webui-vue/files/14190399/1KW_GUI_review.docx)

**Review by SteffiAntony — APPROVED** [review]

**Review by vedangimittal — APPROVED** [review]

**Review by Renuka9527 — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Pull request merged**